app/views/ChangeAvatarView/index.tsx

```diff
@@ -64,6 +65,7 @@ function reducer(state: IState, action: IReducerAction) {
 
 const ChangeAvatarView = () => {
 	const [state, dispatch] = useReducer(reducer, initialState);
+	const [url, setUrl] = useState('');
```

Comment:
Got it. We need the URL state because currently, the Avatar image is updated via the avatar URL only when we click 'Fetch Image URL.' I'll create a rawAvatarUrl property inside the reducer to make it clearer.